From 9cd2e725f4e82da7e5bcab1245141c9a0ede381e Mon Sep 17 00:00:00 2001
From: Jaegeuk Kim <jaegeuk@google.com>
Date: Tue, 7 Aug 2018 12:31:05 -0700
Subject: [PATCH 02/27] vold: Grant root group

When using make_f2fs to format a filesystem with quota enabled, make_f2fs
will write quota record with group id equtal to make_f2fs's execution
group id.
Vold shall run as root group so that make_f2fs forked by vold will enherit
vold's root group. Make_f2fs can then write quota record with group id for
root group and therefore kernel quota driver can find quota record for
root group.

Bug: 111775499
Change-Id: I0c1fef760597d1695586d24365200b25e2cf07e6
Merged-In: I0c1fef760597d1695586d24365200b25e2cf07e6
Signed-off-by: Light Hsieh <light.hsieh@mediatek.com>
Signed-off-by: Jaegeuk Kim <jaegeuk@google.com>
(cherry picked from commit c0eea5736e40eb26a2f88014f73929d84ea7c6b6)
---
 vold.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vold.rc b/vold.rc
index 7d14453..93d8786 100644
--- a/vold.rc
+++ b/vold.rc
@@ -5,4 +5,4 @@ service vold /system/bin/vold \
     ioprio be 2
     writepid /dev/cpuset/foreground/tasks
     shutdown critical
-    group reserved_disk
+    group root reserved_disk
-- 
2.11.0

